tools/coredump/testdata/amd64/ruby-3.5.0-loop.json

```diff
@@ -4,27 +4,26 @@
     {
       "lwp": 1844728,
       "frames": [
-        "is_prime+0 in /app/loop.rb:10",
-        "is_prime+0 in /app/loop.rb:10",
-        "sum_of_primes+0 in /app/loop.rb:20",
-        "<main>+0 in /app/loop.rb:30",
+        "Object#is_prime+0 in /app/loop.rb:0",
         "libruby.so.3.5.0+0x3399b0",
         "libruby.so.3.5.0+0x33e9d7",
         "libruby.so.3.5.0+0x23e175",
         "libruby.so.3.5.0+0x31e2ea",
         "libruby.so.3.5.0+0x325240",
         "libruby.so.3.5.0+0x333b86",
-        "<main>+0 in /app/loop.rb:29",
```

Comment:
This is an example of the repeated frame bug (#770 ) which is now fixed, hence why it doesn't repeat anymore.